### PR TITLE
Add Github stars and last commit info badges

### PR DESCRIPTION
**Pull request by bernie43**
I think it would be a nice addition to see at a glance the number of Github stars and the date of the last commit for each library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,87 +6,170 @@ Read the [contributing guide](/CONTRIBUTING.md)
...
+![Stars](https://img.shields.io/github/stars/dai-shi/react-context-global-state) ![Last Commit](https://img.shields.io/github/last-commit/dai-shi/react-context-global-state)
+- [react-contextual](https://github.com/drcmda/react-contextual) - Is a tiny (less than 1KB) helper around React 16s new context api  
+![Stars](https://img.shields.io/github/stars/drcmda/react-contextual) ![Last Commit](https://img.shields.io/github/last-commit/drcmda/react-contextual)
+- [react-copy-write](https://github.com/aweary/react-copy-write) - Immutable state with a mutable API  
+![Stars](https://img.shields.io/github/stars/aweary/react-copy-write) ![Last Commit](https://img.shields.io/github/last-commit/aweary/react-copy-write)
+- [react-create-context](https://github.com/jamiebuilds/create-react-context) - Polyfill for the proposed React context API.  
+![Stars](https://img.shields.io/github/stars/jamiebuilds/create-react-context) ![Last Commit](https://img.shields.io/github/last-commit/jamiebuilds/create-react-context)
 - [react-easy-state](https://github.com/solkimicreb/react-easy-state) minimal React state management with the power of ES6 Proxies
```
Comment (olegrjumin) [inline README.md]: let's add stars and last commit for react-easy-state too

**Review by olegrjumin — CHANGES_REQUESTED** [review]
Looks perfect, let's update react-easy-state and merge it! Thanks

**Comment by bernie43** [comment]
Here we go 🚀

**Review by olegrjumin — APPROVED** [review]

**Comment by olegrjumin** [comment]
Awesome! 🎉

**Pull request merged**